doc/source/whatsnew/v3.0.0.rst

```diff
@@ -296,6 +296,7 @@ Bug fixes
 - Fixed bug in :meth:`DataFrame.to_string` that raised ``StopIteration`` with nested DataFrames. (:issue:`16098`)
 - Fixed bug in :meth:`DataFrame.update` bool dtype being converted to object (:issue:`55509`)
 - Fixed bug in :meth:`DataFrameGroupBy.apply` that was returning a completely empty DataFrame when all return values of ``func`` were ``None`` instead of returning an empty DataFrame with the original columns and dtypes. (:issue:`57775`)
+- Fixed bug in :meth:`GroupBy.median` where nat values gave an incorrect result. (:issue:`57926`)
```

Comment:
Can you use `` :meth:`.DataFrameGroupBy.median` `` instead - otherwise there wouldn't be a link to the method API docs. Note the `.` prior to `DataFrameGroupBy`.